test/assembler/parser.test.ts

```diff
@@ -34,18 +36,19 @@ describe('parse code', function () {
     expect(ast.areas[1].name).toBe('|.text|')
     expect(ast.areas[1].type).toBe(AreaType.Code)
     expect(ast.areas[1].isReadOnly).toBe(true)
-    expect(ast.areas[1].instructions).toHaveLength(3)
+    expect(ast.areas[1].instructions).toHaveLength(4)
     expect(ast.areas[1].instructions[0].name).toBe('MOVS')
     expect(ast.areas[1].instructions[0].options).toEqual(['R1', '#123'])
     expect(ast.areas[1].instructions[0].line).toBe(8)
     expect(ast.areas[1].instructions[1].name).toBe('MOVS')
     expect(ast.areas[1].instructions[1].label).toBe('label1')
     expect(ast.areas[1].instructions[1].options).toEqual(['R2', '#456'])
     expect(ast.areas[1].instructions[1].line).toBe(9)
-    expect(ast.areas[1].instructions[2].name).toBe('MOVS')
-    expect(ast.areas[1].instructions[2].label).toBe('label2')
-    expect(ast.areas[1].instructions[2].options).toEqual(['R3', '#789'])
-    expect(ast.areas[1].instructions[2].line).toBe(12)
+    expect(ast.areas[1].instructions[2].name).toBe('ALIGN')
```

Comment:
check that there are no options provided (length zero)

Reply:
Changed this.